src/lunamary/model/modelSchool/School.java

```diff
@@ -123,4 +127,13 @@ public void addKardex(Kardex kardex) {
         this.kardexList.add(kardex);
     }
 
+    public HashMap<String, MyLinkedList<Kardex>> getKardexHashMap() {
```

Comment:
I'm not going to decline this PR. Just for this reason but I want you to be aware.
When using java.util.Map you will always pass the interface on method signatures. Concrete implementations can cause issues.